docs/src/Tables.elm

```diff
@@ -1,12 +1,18 @@
 module Tables exposing (stories)
 
 import Element exposing (Element, fill)
+import Msg
+import UI.Internal.TypeNumbers as T
 import UI.RenderConfig exposing (RenderConfig)
 import UI.Table as Table
     exposing
-        ( cellWidthEnd
+        ( cellFromText
+        , cellWidthEnd
         , cellWidthPortion
         , cellWidthShrink
+        , detailsEnd
+        , detailsHide
+        , detailsShow
```

Comment:
Maybe `mobileDetailsShow/Hide/End`, as this module exposes a lot of things, having this prefix makes it more self explanatory